packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -165,6 +171,43 @@ $font-sizes: (
 .c-mediaElement {
     display: flex;
     width: 100%;
+
+    &.c-mediaElement--reverse {
+        flex-direction: row-reverse;
+    }
+}
+
+/**
+ * Modifier – .c-mediaElement--stackMobile
+ *
+ * Applies flex direction column on <=narrow
+ */
+.c-mediaElement--stackMobile {
+    @include media('<=narrow') {
```

Comment:
Does it not work here to stick a single `@include media('<=narrow') { ... }` expression around the `.c-mediaElement--stackMobile { ... }` one, to prevent having to invoke the same mixin multiple times?

Reply:
good point